### DISCUSSION TITLE
Content and behavior addressable code

### DISCUSSION THREAD

**Discussion by SeanPedersen** [python-poetry/poetry#4950] [Ideas]
- [x] I have searched the [issues](https://github.com/python-poetry/poetry/issues) of this repo and believe that this is not a duplicate.
- [x] I have searched the [documentation](https://python-poetry.org/docs/) and believe that my question is not covered.

## Issue
I love poetry for its slick and intuitive CLI but IMO it lacks some major features that will be the future of programming package management. In particular content and behavior addressable code, read more about it here: https://blog.neotree.uber.space/posts/future-of-programming

What do you think about these concepts? Could they move poetry forward?

**Comment by sinoroc** [comment]
It does not seem to have much to do with poetry. At least maybe move this to discussions: https://github.com/python-poetry/poetry/discussions
